README.md

```diff
@@ -1 +1,29 @@
-# guesthouse
\ No newline at end of file
+# Guesthouse
+
+A repository containing information and documentation for the **FACN2** guesthouse.
+
+## House Rules
+
+**Follow Founders & Coders Code of Conduct:**
+
+Our community is dedicated to providing a harassment-free experience for everyone, regardless of gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion (or lack thereof), or technology choices. We do not tolerate harassment of community members in any form. Sexual language, imagery or innuendo is not appropriate for any community event, including talks, workshops, parties, Gitter, Twitter and other online media or channels. Participants violating these rules may be sanctioned or expelled from the event and any future events, as well as the from the community itself, at the discretion of the organisers, Founders & Coders.
+
+## Communal living
+
+Communal living means respecting other people all the time.
+
+### Cleaning rotation
+There are scheduled cleanings twice a month done by professional cleaners. All the other weeks there is a **cleaning rotation** for all  communal areas.
```

Comment:
@matthewdking I can get you an answer on Monday, cause i haven't talked to Tareq for awhile